### PR TITLE
add example of a user-entered tolerance; emphasize Test is a core module

### PR DESCRIPTION
**Pull request by tbrowder**
Add a note about Test being a core module.  Also add an example of a user-supplied tolerance for numerical testing.

I find these additions helpful. It isn't clear in current docs that Test is a core module, and thus a casual user has to search a while for its source.  A section on core modules would be helpful, perhaps referenced on the page where the external modules are listed: <https://modules.perl6.org/>.

Another place to reference them would be to add another category, "Core Modules," on this page: <https://doc.perl6.org/>

### CODE REVIEW

**Comment by pmichaud** [comment]
Can we please not call modules like Test.pm "core modules"?   The term "CORE" already has a very specific meaning in Perl 6 (and Test.pm is not a part of CORE).

S24 refers to Test.pm as being a "standard module" -- that phrasing would sit much better with me.

Pm

**Comment by tbrowder** [comment]
On Sunday, April 17, 2016, Patrick R. Michaud <notifications@github.com>
wrote:

> Can we please not call modules like Test.pm "core modules"? The term
> "CORE" already has a very specific meaning in Perl 6 (and Test.pm is not a
> part of CORE).
>
> S24 refers to Test.pm as being a "standard module" -- that phrasing would
> sit much better with me.
>

Sure, that's fine with me. I was just using Perl 5 lingo I guess. The point
is that the casual user will not easily find the docs for the module.

-Tom

>

**Comment by tbrowder** [comment]
On Sunday, April 17, 2016, Tom Browder <tom.browder@gmail.com> wrote:

>
>
> On Sunday, April 17, 2016, Patrick R. Michaud <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> Can we please not call modules like Test.pm "core modules"? The term
>> "CORE" already has a very sp
>>
>

> S24 refers to Test.pm as being a "standard module" -- that phrasing would
>> sit much better with me.
>>
>
> Sure, that's fine with me. I was just using Perl 5 lingo I guess. The
> point is that the casual user will not easily find the docs for the module.
>

I plan to prepare a pull request for the website soon that I think will
help the situation.

-Tom

>

**Comment by stmuk** [comment]
Maybe we need a glossary of Perl 5 terms to Perl 6 terms?

File: doc/Language/testing.pod [hunk 1]
```diff
@@ -141,6 +141,12 @@ C<$description> of the test.
     my $eulers-constant-approx = 2.71828;
     is-approx $eulers-constant-approx, e, "approximate Euler's constant";
 
+The use can also input a desired tolerance, e.g., testing $some-number:
+
+    my $spec-num = 1.2345;
+    my $spec-tol = 0.1;
+    is-approx $some-number, $spec-num, $tol, "special tolerance check";
```
Comment (MadcapJake) [inline doc/Language/testing.pod]: `Variable '$tol' is not declared. Did you mean '$spec-tol'?`

Comment (tbrowder) [inline doc/Language/testing.pod]: On Sun, Apr 17, 2016 at 1:18 PM, Jake Russo <notifications@github.com> wrote:
>
> In doc/Language/testing.pod:
...
> > +    my $spec-tol = 0.1;
> > +    is-approx $some-number, $spec-num, $tol, "special tolerance check";
>
> Variable '$tol' is not declared. Did you mean '$spec-tol'?


Yes!  Good catch--sorry.

-Tom

**Comment by tbrowder** [comment]
I just updated the commits to reflect comments in this conversation.

**Comment by tbrowder** [comment]
I could merge this request, but am reluctant to do so without a more experienced dev's approval.

**Comment by zoffixznet** [comment]
@tbrowder It's not at all clear how to use the tolerance argument. What's the meaning of value `0.1`? I'd expand on that.

Also, there's a typo: `The use can also` should probably be **user**.

As for merging... unless it's a huge change that modifies a lot of content, just merge or commit directly to master. If there are any issues, they can be fixed in another commit :smile_cat:

**Pull request merged**